deployment/helm/balloons/crds/config.nri_balloonspolicies.yaml

```diff
@@ -513,6 +520,18 @@ spec:
                   type: string
                 description: Reserved (CPU) resources for kube-system namespace.
                 type: object
+              showContainersInNrt:
+                description: |-
+                  ShowContainersInNrt controls whether containers in balloons
```

Comment:
nit: Same here, especially since this is the global default... So to address the concerns raised by @fmuyassarov, shouldn't we add here a word of warning about how enabling this will affect the amount of data pushed to the CR / API server from nodes with a large amount of containers ?